FIRMWARE/PocketSDVX/PocketSDVX.ino

```diff
@@ -92,30 +115,39 @@ void setup() {
   // Startup mode
   int Button1State = digitalRead(BT_A); //Read Btn-A
   int Button2State = digitalRead(BT_B); //Read Btn-B
-  // Button 1 is held down: Joystick Mode
-  if (Button1State == LOW && Button2State == HIGH) {
-    if (currMode != 1) {
-      EEPROM.update(0, 1); 
-      delay(200);
-    }
-  } else if (Button2State == LOW && Button1State == HIGH) {
-    // Button 2 is held down: Keyboard Mode
-    if (currMode != 2) {
-      EEPROM.update(0, 2);
-      delay(200);
-    }
+  int Button3State = digitalRead(BT_C); //Read Btn-C
+
+  bool IsButtonAPressed = Button1State == LOW;
+  bool IsButtonBPressed = Button2State == LOW;
+  bool IsButtonCPressed = Button3State == LOW;
+
+  bool UseJoystick = IsButtonAPressed && !IsButtonBPressed;
+  bool UseKeyboard = IsButtonBPressed && !IsButtonAPressed;
+  bool UseLight = !IsButtonCPressed;
+
+  if (UseJoystick) {
+    updateEEPROM(EEPROM_CONTROL_ADDRESS, MODE_JOYSTICK);
+  } else if (UseKeyboard) {
+    updateEEPROM(EEPROM_CONTROL_ADDRESS, MODE_KEYBOARD);
+  } else {
+    updateEEPROM(EEPROM_CONTROL_ADDRESS, MODE_JOYSTICK);
+  }
+
+  if (UseLight) {
+    updateEEPROM(EEPROM_LIGHT_ADDRESS, MODE_LIGHT_ON);
+  } else {
+    updateEEPROM(EEPROM_LIGHT_ADDRESS, MODE_LIGHT_OFF);
```

Comment:
Refactored this as well :D

If C button is pressed then update EEPROM address to LIGHT_OFF, then later in the lights() function it will check if the light address is on or off and determine

Reply:
Note that it's independent of control mode

So if (B + C) is pressed -> Keyboard mode and lightless

Joystick with light on by default